consul/operator_endpoint.go

```diff
@@ -183,3 +184,63 @@ func (op *Operator) AutopilotSetConfiguration(args *structs.AutopilotSetConfigRe
 	}
 	return nil
 }
+
+// Used by Autopilot to query the raft stats of the local server.
+func (op *Operator) RaftStats(args struct{}, reply *map[string]string) error {
```

Comment:
This might be good to put into the `Status` endpoint - that more naturally is "things that don't require an ACL", and we may mix non-Raft things in here in the future.